shub/tool.py

```diff
@@ -24,6 +24,8 @@ def cli():
     "deploy_reqs": [],
     "logout": [],
     "version": [],
+    "items": ["hubstorage"],
```

Comment:
hubstorage should be a required dependency, and shipped with shub tool.

These commands should always work, not like deploy that requires scrapy to be present.